tests/Microsoft.Identity.Test.Unit.net45/PublicApiTests/ClientCredentialWithCertTest.cs

```diff
@@ -203,9 +203,8 @@ public async Task JsonWebTokenWithX509PublicCertSendCertificateSilentTestAsync()
                 harness.HttpManager.AddMockHandler(CreateTokenResponseHttpHandlerWithX5CValidation(false));
 
                 var result = await app
-                    .AcquireTokenSilent(
-                        new[] { "someTestScope"},
-                        new Account(MsalTestConstants.UserIdentifier, MsalTestConstants.DisplayableId, null))
+                    .AquireTokenSilentWithSingleAccount(
```

Comment:
>AquireTokenSilentWithSingleAccount [](start = 21, length = 34)

not sure this was the original intention of the test, but suppose it's an ok way of validating

Reply:
not entirely but it does make the test simpler. an additional unit test for AquireTokenSilentWithSingleAccount() will be added